sageintacctsdk/apis/api_base.py

```diff
@@ -231,7 +232,10 @@ def __post_request(self, dict_body: dict, api_url: str):
                 parsed_response = parsed_response['response']['errormessage']
 
         if raw_response.status_code == 400:
-            raise WrongParamsError('Some of the parameters are wrong', parsed_response)
+            if 'invalidRequest' == parsed_response['error']['errorno']:
```

Comment:
safely compare this
`if 'error' in parsed_response and 'errorno' in parsed_response['error'] and parsed_response['error']['errorno'] == 'invalidRequest'`

Also, there is one more key in top level to check. Check the response from intacct API. some status is returned as failure.